traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -984,6 +984,11 @@ func Update(w http.ResponseWriter, r *http.Request) {
 	}
 	defer inf.Close()
 
+	//Get original xmppid
+	origSer, _, _, _, _, _ := getServers(r.Header, inf.Params, inf.Tx, inf.User, false)
```

Comment:
We should check for the errors here, and only dereference the `origSer` variable in the case when there was no error

Reply:
Added error check